include/ppconsul/kv.h

```diff
@@ -329,15 +332,95 @@ namespace ppconsul { namespace kv {
         std::string m_defaultDc;
     };
 
-    // TODO: add ScopedLock
-
-    // Implementation
-
     namespace impl {
         StringList parseKeys(const std::string& resp);
         std::vector<KeyValue> parseValues(const std::string& resp);
+
+        namespace session {
+            std::string createBodyJson(const std::string &node, int lockDelay,
+                                       const std::string &behavior, int ttl);
+            std::string parseCreateResponse(const std::string &resp);
+        }
     }
 
+    class Sessions
+    {
+    public:
+        enum class OnInvalidation
+        {
+            RELEASE,
```

Comment:
Don't use all capital letters for anything that's not a macro. Use `CamelCase` instead.